tests/unit/src/test/scala/tests/StatusBarLspSuite.scala

```diff
@@ -36,7 +38,7 @@ class StatusBarLspSuite extends BaseLspSuite("status-bar") {
         s"${icons.alert}Compiled a",
       )
       _ <- server.didSave("a/src/main/scala/A.scala")(
-        _.replaceFirst("val x: String = 42", "val x = 42")
+        _.replaceFirst("val x: String = 42", "val x: Long = 42")
```

Comment:
This was more tricky, since when compilation fails we don't actually copy over compilation artifacts, so if we get back to previous state no actual compilation is done.